scripts/upgradeNocodbSdk.js

```diff
@@ -22,7 +22,8 @@ const replacePackageVersion = (filePath) => {
     return new Promise((resolve, reject) => {
         return fs.readFile(filePath, 'utf8', function (err, data) {
             if (err) return reject(err)
-            var result = data.replace(/workspace:\^/g, nocodbSdkPackage.version);
+            const rgx = new RegExp(`"${nocodbSdkPackage.name}": "(.*)"`, "g");
```

Comment:
No need for the capturing group here since we are not using it.


Following is enough
```javascript
 const rgx = new RegExp(`"${nocodbSdkPackage.name}": ".*"`, "g");
```